nastaveni/nastaveni.php

```diff
@@ -191,7 +191,8 @@
 if (!defined('DEN_PRVNI_DATE')) define('DEN_PRVNI_DATE', date('Y-m-d', strtotime(PROGRAM_OD))); // první den v programu ve formátu YYYY-MM-DD
 if (!defined('DEN_PRVNI_UBYTOVANI')) define('DEN_PRVNI_UBYTOVANI', DEN_PRVNI_DATE); // datum, kterému odpovídá ubytovani_den (tabulka shop_predmety) v hodnotě 0
 if (!defined('PROGRAM_ZACATEK')) define('PROGRAM_ZACATEK', 8); // první hodina programu
-if (!defined('PROGRAM_KONEC')) define('PROGRAM_KONEC', 24); // konec programu (tuto hodinu už se nehraje)
+if (!defined('PROGRAM_KONEC')) define('PROGRAM_KONEC', 6); // konec programu (tuto hodinu už se nehraje). Používejte pouze hodnoty 1-24. Pokud je menší než PROGRAM_ZACATEK, znamená to, že končí následujícího dne.
+if (PROGRAM_KONEC === 0) define('PROGRAM_KONEC', 24); // kontrola kdyby někdo zadal konec jako 0 (rozbíjí to vytváření aktivit, noční aktivity aj.)
```

Comment:
Tohle jsi myslel dobře, ale bohužel to v PHP nelze.
Jak jednou definuješ konstantu, tak už s ní nehneš:
```
php -a
Interactive shell

php > define('LADA_DRTI', 'js');
php > if (LADA_DRTI !== 'php') define('LADA_DRTI', 'php');
PHP Warning:  Constant LADA_DRTI already defined in php shell code on line 1
PHP Stack trace:
PHP   1. {main}() php shell code:0
PHP   2. define($constant_name = 'LADA_DRTI', $value = 'php') php shell code:1
php > echo LADA_DRTI;
js
```
Asi bych tu natvrdo hodil exception (tohle nejde nastavit jinde než přes PHP, takže ať si to programátor po sobě opraví).